sales_team_operating_unit/tests/test_crm_team_operating_unit.py

```diff
@@ -47,6 +47,7 @@ def _create_user(self, login, groups, company, operating_units,
             'email': 'test@yourcompany.com',
             'company_id': company.id,
             'company_ids': [(4, company.id)],
+            'default_operating_unit_id': self.ou1.id,
```

Comment:
I think it is more logical to assign the first operating unit passed to that method: operating_units[0].id